### PR TITLE
Expand tests to image helper

### PR DESCRIPTION
**Pull request by chriseyre2000**
We now have unit tests for the image helper.

These changes included adding a validation rule that will fail the build if the alt text is empty or missing.

### CODE REVIEW

**Comment by chriseyre2000** [comment]
Currently these images do not work on the preview site.
I am going to fix this.

This is the problem article: 2017-06-04-ncraft-in-20-pictures.md

File: spec/_plugins/image_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    it 'Can render a simple image with alt text' do
+      token = double
+      expect(token).to receive :line_number
+
+      tag = Jekyll::ImageTag.send :new, 'img', '/assets/custom/img/blog/ncraft-2017/event-storming-with-brandolini.jpg "ALT"', token
+
+      expect(tag.render(nil)).to eq('<p></p><img src="/assets/custom/img/blog/ncraft-2017/event-storming-with-brandolini.jpg"  alt="ALT" title="ALT" class="img img-center img-fluid style-screengrab">')
+    end
+    
+    it 'will throw an exception when an img does not have a non-blank alt tag' do
```
Comment (alfredodev) [inline spec/_plugins/image_helper_spec.rb]: double negation here "does not have a non-blank alt"
What do you think about this? 
"will throw an exception when an img has a blank alt tag"

**Comment by alfredodev** [comment]
Nice alt messages!

File: spec/_plugins/image_helper_spec.rb (hunk 1 above)
Comment (chriseyre2000) [reply spec/_plugins/image_helper_spec.rb]: Renamed.

**Pull request merged**